assets/css/calypsoify.css

```diff
@@ -131,12 +131,134 @@ div.woocommerce-message, .wc-helper .start-container {
+	background-size: contain;
+}
+
+.wc-setup label {
+	color: #2E4453;
+	font-size: 14px;
+	font-weight: 600;
+}
+
+/* Onboarding wizard select2 */
+.wc-setup .select2 {
+	max-width: 100%;
+	margin-bottom: 13px;
+}
+.wc-setup .select2-container .select2-selection {
+	border-color: #C8D7E1;
+	box-shadow: 0 1px 0 #C8D7E1;
+}
+.wc-setup .select2-container .select2-selection--single {
+    height: 39px;
+}
+.wc-setup .select2-container .select2-selection--single .select2-selection__rendered {
+	color: #2E4453;
+	line-height: 38px;
+	padding-right: 36px;
+	font-size: 14px;
+	font-weight: 600;
+	padding-left: 16px;
+}
+.select2-container .select2-selection--single .select2-selection__arrow {
+    right: 11px;
+	height: 39px;
+	width: 24px;
+}
+.wc-setup .select2-container .select2-selection__arrow:before {
+	content: '\f347';
+	font-family: dashicons;
+	line-height: 38px;
+	color: #C8D7E1;
+	font-size: 24px;
+}
+.wc-setup .select2-container .select2-selection__arrow b {
+	display: none;
+}
+
+/* Onboarding wizard store setup */
+.store-setup {
+	display: none;
+}
+.store-address-container label:first-child {
+	margin-top: 0;
+}
+.location-prompt {
+	margin-top: 10px;
+}
+.store-address-container .city-and-postcode {
+    margin-bottom: 20px;
+}
+.product-type-container {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+#store_address_2 {
+	display: none;
```

Comment:
Looks like a little bit of indent oddness going on here and down in the next selector too.

Reply:
Thanks!  Didn't show up in my editor at all for some reason.  Reindented and pushed!